registration/models.py

```diff
@@ -376,3 +377,364 @@ def send_activation_email(self, site, request=None):
+
+            with transaction.atomic():
+                profile.user.save()
+
+            profile.send_admin_approve_complete_email(site, request)
+
+            if get_profile:
+                return profile
+            else:
+                return profile.user
+        except self.model.DoesNotExist:
+            return False
+
+    def send_admin_approve_email(self, user, site, request=None):
+
+        """
+        Send an approval email to the site administrators to
+        approve this user.
+
+        The approval email will use the following templates,
+        which can be overriden by setting APPROVAL_EMAIL_SUBJECT,
+        APPROVAL_EMAIL_BODY, and APPROVAL_EMAIL_HTML appropriately:
+
+        ``registration/admin_approve_email_subject.txt``
+            This template will be used for the subject line of the
+            email. Because it is used as the subject line of an email,
+            this template's output **must** be only a single line of
+            text; output longer than one line will be forcibly joined
+            into only a single line.
+
+        ``registration/admin_approve_email.txt``
+            This template will be used for the text body of the email.
+
+        ``registration/admin_approve_email.html``
+            This template will be used for the html body of the email.
+
+        These templates will each receive the following context
+        variables:
+
+        ``user``
+            The new user account
+
+        ``user_id``
+            The id of the associated``SupervisedRegistrationProfile``
+            object.
+
+        ``site``
+            An object representing the site on which the user
+            registered; depending on whether ``django.contrib.sites``
+            is installed, this may be an instance of either
+            ``django.contrib.sites.models.Site`` (if the sites
+            application is installed) or
+            ``django.contrib.sites.requests.RequestSite`` (if
+            not). Consult the documentation for the Django sites
+            framework for details regarding these objects' interfaces.
+
+        ``request``
+            Optional Django's ``HttpRequest`` object from view.
+            If supplied will be passed to the template for better
+            flexibility via ``RequestContext``.
+        """
+
+        admin_approve_email_subject = getattr(
+            settings,
+            'ADMIN_APPROVAL_EMAIL_SUBJECT',
+            'registration/admin_approve_email_subject.txt'
+        )
+        admin_approve_email_body = getattr(
+            settings,
+            'ADMIN_APPROVAL_EMAIL_BODY',
+            'registration/admin_approve_email.txt'
+        )
+        admin_approve_email_html = getattr(
+            settings,
+            'ADMIN_APPROVAL_EMAIL_HTML',
+            'registration/admin_approve_email.html'
+        )
+
+        ctx_dict = {}
+        if request is not None:
+            ctx_dict = RequestContext(request, ctx_dict)
```

Comment:
Follow the pattern here so we don't need to use `RequestContext` anymore https://github.com/macropin/django-registration/pull/194